homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -206,6 +216,10 @@ def media_image_url(self):
             self._source = await fs_device.get_mode()
             self._mute = await fs_device.get_mute()
             self._media_image_url = await fs_device.get_play_graphic()
+
+            volume = await self.fs_device.get_volume()
+
+            self._volume_level = float(volume or 0) / int(self._volume_steps)
```

Comment:
Isn't `self._volume_steps` always an `int`?
```suggestion
            self._volume_level = float(volume or 0) / self._volume_steps
```